packages/datadog-instrumentations/src/lodash.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { channel, addHook } = require('./helpers/instrument')
+
+const shimmer = require('../../datadog-shimmer')
+
+addHook({ name: 'lodash', versions: ['>=4'] }, lodash => {
+  const lodashOperationCh = channel('datadog:lodash:operation')
+
+  const instrumentedLodashFn = ['trim', 'trimStart', 'trimEnd', 'toLower', 'toUpper', 'join']
+
+  shimmer.massWrap(
+    lodash,
+    instrumentedLodashFn,
+    lodashFn => {
+      return function () {
+        if (!lodashOperationCh.hasSubscribers) {
+          return lodashFn(...arguments)
+        }
+
+        const result = lodashFn(...arguments)
+        lodashOperationCh.publish({ operation: lodashFn.name, arguments, result })
+
+        return result
```

Comment:
```suggestion
        const result = lodashFn(...arguments)
        if (result && lodashOperationCh.hasSubscribers) {
          lodashOperationCh.publish({ operation: lodashFn.name, arguments, result })
        }
        return result
```

Reply:
Avoid publishing the message would couple lodash instrumentation to taint tracking system. I would prefer to add a check in taintTrackingHandler to prevent calling taint method if result is an empty string. WDYT?